aixplain/factories/pipeline_factory/__init__.py

```diff
@@ -220,23 +227,34 @@ def list(
             payload["inputDataTypes"] = [data_type.value for data_type in output_data_types]
 
         logging.info(f"Start service for POST List Pipeline - {url} - {headers} - {json.dumps(payload)}")
-        r = _request_with_retry("post", url, headers=headers, json=payload)
-        resp = r.json()
-
-        pipelines, page_total, total = [], 0, 0
-        if "items" in resp:
-            results = resp["items"]
-            page_total = resp["pageTotal"]
-            total = resp["total"]
-            logging.info(f"Response for POST List Pipeline - Page Total: {page_total} / Total: {total}")
-            for pipeline in results:
-                pipelines.append(build_from_response(pipeline))
-        return {
-            "results": pipelines,
-            "page_total": page_total,
-            "page_number": page_number,
-            "total": total,
-        }
+        try:
+            r = _request_with_retry("post", url, headers=headers, json=payload)
+            resp = r.json()
+            if 200 <= r.status_code < 300:
```

Comment:
This condition should be outside the try catch. Exactly how it is in the API Key factory.